priv/repo/migrations/20190501164331_create_address_history_procedure.exs

```diff
@@ -0,0 +1,37 @@
+defmodule App.Repo.Migrations.CreateAddressHistoryProcedure do
+  use Ecto.Migration
+
+  def up do
+    execute """
+    CREATE OR REPLACE FUNCTION notify_address_changes()
+    RETURNS trigger AS $$
+    BEGIN
+      PERFORM pg_notify(
+        'addresses_changed',
+        json_build_object(
+          'operation', TG_OP,
+          'record', row_to_json(NEW)
+        )::text
+      );
+      RETURN NEW;
+    END;
+    $$ LANGUAGE plpgsql;
+    """
+
+    flush()
```

Comment:
["_courtesy_ `flush`?"](https://youtu.be/J-YVt4gfquA?t=76) 💩 
joking aside, comments would be _really_ useful in this file. 😉